samples/bidirectional-chat/csharp/Function.cs

```diff
@@ -80,8 +78,8 @@ public async Task LeaveUserFromGroup([SignalRTrigger]InvocationContext invocatio
             await UserGroups.RemoveFromGroupAsync(userName, groupName);
         }
 
-        [FunctionName(nameof(Disconnect))]
-        public void Disconnect([SignalRTrigger]InvocationContext invocationContext)
+        [FunctionName(nameof(OnDisconnect))]
+        public void OnDisconnect([SignalRTrigger]InvocationContext invocationContext)
```

Comment:
Why not OnDisconnected...? it doesn't consistent to `OnConnected`

Reply:
Oh, nice catch.